addons/payment/models/payment_transaction.py

```diff
@@ -910,6 +912,19 @@ def _execute_callback(self):
 
     #=== BUSINESS METHODS - POST-PROCESSING ===#
 
+    def _str_to_datetime(self, value):
```

Comment:
Any appropriate arguments can be added. I passed the value forcefully so that it can be easily used in the future.
[reference file](https://github.com/odoo/odoo/blob/master/odoo/addons/base/models/ir_fields.py)
 
:+1: method name changed.